### PR TITLE
Added auth feature & sidebar

### PR DESCRIPTION
**Pull request by parthgabani777**
- Added login & signup features
- Created sidebars

### CODE REVIEW

File: src/components/sidebar/left-sidebar.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+.sidebar a {
+    color: var(--white-color);
+    border-radius: 1.5rem;
+    padding: 1rem 1.5rem;
+    margin: 1rem;
+}
+.sidebar a:hover {
+    background: var(--primary-color);
+    /* color: var(--white-color); */
```
Comment (nerdyspook) [inline src/components/sidebar/left-sidebar.css]: You can remove this comment from here if not required.

**Review by nerdyspook — APPROVED** [review]
The UI looks pretty good, and the loader is working as well.

**Review by shivanigangadharan — APPROVED** [review]
Code looks good, functionality also working well in preview.

File: src/App.js [hunk 2]
```diff
@@ -1,43 +1,19 @@
 import "./App.css";
-import logo from "./logo.png";
+import "./css/design.css";
+import { RightSidebar } from "./components/sidebar/right-sidebar";
+import { LeftSidebar } from "./components/sidebar/left-sidebar";
```
Comment (SaurabhChawda) [inline src/App.js:4]: Use index Files for import and export.

File: src/components/sidebar/right-sidebar.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import "./right-sidebar.css";
+import { Link } from "react-router-dom";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+export function RightSidebar() {
+    const dispatch = useDispatch();
+    const { isAuthorized, token } = useSelector((state) => state.authReducer);
+
+    useEffect(() => {}, []);
```
Comment (SaurabhChawda) [inline src/components/sidebar/right-sidebar.js:10]: is it good to useEffect like this?

**Review by SaurabhChawda — APPROVED** [review]
Overall looks good, You can merge it.

File: src/components/sidebar/right-sidebar.js (hunk 3 above)
Comment (parthgabani777) [reply src/components/sidebar/right-sidebar.js:10]: I was gonna use it in future

**Pull request merged**